packages/pcd/gpc-pcd/src/GPCPCD.ts

```diff
@@ -33,7 +39,17 @@ export type GPCPCDInitArgs = {
 };
 
 /**
- * Defines the essential parameters required for creating a {@link GPCPCD}.
+ * Record argument type encapsulating a record of POD PCD arguments.
```

Comment:
Discuss that the names need to correspond to the names used in the proof config.